api/controller.py

```diff
@@ -1167,7 +1167,7 @@ def search(self, lane_identifier, feed_class=AcquisitionFeed):
         if isinstance(lane, ProblemDetail):
             return lane
 
-        # Althoug the search query goes against Elasticsearch, we must
+        # Althoug the search query goes against Opensearch, we must
```

Comment:
Missing `h` on `Although`. Obviously not your fault, but given that you're already updating comments...